library/src/com/google/maps/android/geojson/GeoJsonLayer.java

```diff
@@ -0,0 +1,270 @@
+package com.google.maps.android.geojson;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.LatLngBounds;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import android.content.Context;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+
+/**
+ * A class that allows the developer import GeoJSON data, style it and interact with the imported
+ * data.
+ *
+ * To create a new GeoJsonLayer from a resource stored locally
+ * {@code <pre>GeoJsonLayer layer = new GeoJsonLayer(getMap(), R.raw.resource, getApplicationContext());
```

Comment:
So for all of these {@code} blocks, if you want it inline then drop the &lt;pre&gt; tags, if you want it in it's own section then put the tags outside of the {@code} blocks (and remember to close them)